tfhe/src/core_crypto/algorithms/seeded_ggsw_ciphertext_decompression.rs

```diff
@@ -23,27 +24,22 @@ pub fn decompress_seeded_ggsw_ciphertext_with_existing_generator<
     OutputCont: ContainerMut<Element = Scalar>,
     Gen: ByteRandomGenerator,
 {
-    // Generator forking must match the encryption algorithm
-    let output_decomp_level_count = output_ggsw.decomposition_level_count();
-    let output_glwe_size = output_ggsw.glwe_size();
-    let output_polynomial_size = output_ggsw.polynomial_size();
+    let ggsw_forking_configuration = input_seeded_ggsw.decompression_fork_config(Uniform);
```

Comment:
The config is built from the input but the output was previously used
Is this expected?
Comment applies to other files

Reply:
yes it does, basically I decided to put the decompression config on the seeded entity, the input and output are supposed to be compatible so the metadata is the same between them